scripts/configure_dev.sh

```diff
@@ -75,7 +75,7 @@ if [ "${OS}" = "linux" ]; then
     fi
 
     sudo apt-get update
-    sudo apt-get install -y libboost-all-dev expect jq autoconf shellcheck sqlite3 python3-venv
+    sudo apt-get install -y libboost-all-dev expect jq autoconf shellcheck sqlite3 python3-venv make g++
```

Comment:
are these missing on vanilla Ubuntu 18.04 ? If not, we need to branch.

Reply:
Wouldn't `build-essential` be preferable here?